services/src/main/java/org/fao/geonet/api/records/formatters/ImageReplacedElementFactory.java

```diff
@@ -237,7 +240,9 @@ public UrlImageLoader(String url) {
 
         @Override
         public Image loadImage() throws Exception {
-            Log.error(Geonet.GEONETWORK, "URL -> " + url);
+            URI normalizedUrl = new URI(url).normalize();
+            Log.debug(Geonet.GEONETWORK, String.format("URL -> %s, normalized URL -> %s",
+                url, normalizedUrl.toString()));
 
             try (InputStream input = new URL(url).openStream()) {
```

Comment:
Was normalizedUrl just added for the logs?
Should we be using `normalizedUrl.openStream()` here?

Reply:
@pmauduit can you check this and see if @ianwallen proposal is correct for your fix? Thanks 
I suppose you get this in geOrchestra - did you report here the complete fix?

Reply:
Checking what i did in the georchestra fork, you are correct, I don't know what I missed here, because I generally create a branch & cherry-pick my commits when I want to backport it, but I might have had a conflict solved in a wrong way ... see https://github.com/georchestra/geonetwork/commit/0352d0f6f4ea932061d64d068aa5e8c6c3451942 sorry for this

Reply:
@pmauduit I tested the change using the code from https://github.com/georchestra/geonetwork/commit/0352d0f6f4ea932061d64d068aa5e8c6c3451942, but apart of the url normalisation, what looks good, I don't see any difference between the PDF created without or with the url normalised.

Could be this caused by any http proxy or firewall configuration?

Reply:
No, this is probably because the spring / spring security version in GeoNetwork is older than the one that we use in geOrchestra: If we ask for a non-normalized URL, it won't get through the geOrchestra proxy (based on spring / spring security, but in a newer version than the one used in geonetwork currently), because non-normalized urls will be rejected.

It probably still works in a vanilla GeoNetwork, but will fail after the next spring / spring security upgrade (because of new filters in the spring chain that disallow non-normalized urls). That is what I intended to explain in the PR message.

Reply:
Thanks @pmauduit  for the clarification. If you can update the PR with the correct code, I'll merge it.

Reply:
sure, sorry for the mistake in the first place, thanks @ianwallen for having caught it

Reply:
Ok, rebased